docs/learn/learn-runtime-upgrades.md

```diff
@@ -10,35 +10,24 @@ slug: ../learn-runtime-upgrades
 
 Runtime upgrades allow Polkadot to change the logic of the chain, without the need for a hard fork.
 
-## Forking vs Forkless
+## Forkless Upgrades
 
-When we talk about "forking", we are referring to the act of invoking the `fork` system call [within
-a system], where a process creates a copy of itself. In software development, this is the act of
-forking source code from one software package that creates a separate piece of that software.
+You may have come across the term "hard fork" before in the blockchain space. A **hard fork** occurs
+when a blockchain's logic changes such that nodes that do not include the new changes will not be
+able to remain in consensus with nodes that do. Such changes are backwards incompatible. Hard forks
```

Comment:
"backwards incompatible" ->"backward incompatible"

Reply:
Both are equally acceptable according to modern style guides.